ra/ra.go

```diff
@@ -55,6 +55,16 @@ import (
 	"github.com/letsencrypt/boulder/web"
 )
 
+const (
+	// allowed is used for rate limit metrics, it's the value of the 'decision'
+	// label when a request was allowed.
+	allowed = "allowed"
+
+	// denied is used for rate limit metrics, it's the value of the 'decision'
+	// label when a request was denied.
+	denied = "denied"
```

Comment:
Any particular reason to change from the existing "pass" or "exceeded" terminology?

Reply:
'allowed' and 'denied' match the terminology used in the new key-value ratelimits package. I'm attempting to match as much of the terminology as I can.